projects/pastanaga/src/lib/textfield/textfield.common.ts

```diff
@@ -155,7 +155,7 @@ export class TextfieldCommon implements ControlValueAccessor, OnInit, Validator
         this.onChange = (handler as Function);
     }
 
-    setDisabledState(disabled: boolean) {
+    setDisabledState(disabled: boolean | undefined) {
```

Comment:
can't be undefined as we're using `coerceBoolean`.
Also, not your code but the line below is wrong as well: should be `this._disabled = disabled;`